### ISSUE TITLE
ImportError: cannot import name 'transfer' from 'giftless'

### ISSUE THREAD

**Issue by ANaumann85** [datopian/giftless#56]
python version: 3.7.3
pip3 version: 18.1
giftless version: 0.2.0

When running from Pypi with
```
uwsgi -M -T --threads 2 -p 2 --manage-script-name \
    --module giftless.wsgi_entrypoint --callable app --http 127.0.0.1:8080
```
I get the error `ImportError: cannot import name 'transfer' from 'giftless' `.

It seems like the folder "transfer" is missing in the pypi module.

**Comment by shevron** [comment]
Thanks for reporting, I've confirmed this and will work to fix.

**Comment by shevron** [comment]
This should be resolved in 0.2.1 - please try it.
